src/scripts/todolist.js

```diff
@@ -0,0 +1,167 @@
+      'm-0',
+      'container',
+      'fs-5',
+    );
+    taskDescription.innerHTML = wee.description;
+    if (wee.completed === true) {
+      checkbox.checked = true;
+      taskDescription.classList.add('text-decoration-line-through');
+    } else {
+      checkbox.checked = false;
+      taskDescription.classList.add('text-decoration-none');
+    }
+
+    const dots = document.createElement('span');
+    dots.classList.add('dots');
+    dots.innerHTML = `<svg class="text-secondary text-opacity-10" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-three-dots-vertical" viewBox="0 0 16 16">
+        <path d="M9.5 13a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z"/>
+      </svg>`;
+
+    const bin = document.createElement('span');
+    bin.classList.add('d-none', 'bin');
+    bin.innerHTML = `<svg class="text-secondary text-opacity-10" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-trash" viewBox="0 0 16 16">
+        <path d="M5.5 5.5A.5.5 0 0 1 6 6v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm2.5 0a.5.5 0 0 1 .5.5v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm3 .5a.5.5 0 0 0-1 0v6a.5.5 0 0 0 1 0V6z"/>
+        <path fill-rule="evenodd" d="M14.5 3a1 1 0 0 1-1 1H13v9a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V4h-.5a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1H6a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1h3.5a1 1 0 0 1 1 1v1zM4.118 4 4 4.059V13a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V4.059L11.882 4H4.118zM2.5 3V2h11v1h-11z"/>
+      </svg>`;
+
+    taskElement.append(checkbox, taskDescription, dots, bin);
+    taskHolder.append(taskElement);
+  }
+
+  // show task
+
+  showTask() {
+    this.taskInfo.forEach((tasks) => {
+      TodolistClass.displayTask(tasks);
+    });
+  }
+
+  // updateTaskStatus
+  updateTaskStatus(chck, index) {
+    chck.addEventListener('click', (r) => {
+      if (r.currentTarget.checked) {
+        chck.nextElementSibling.classList.add('text-decoration-line-through');
+        this.taskInfo[index].completed = true;
+        localStorage.setItem('Todo', JSON.stringify(this.taskInfo));
+      } else {
+        if (
+          chck.nextElementSibling.classList.contains(
+            'text-decoration-line-through',
+          )
+        ) {
+          chck.nextElementSibling.classList.remove(
+            'text-decoration-line-through',
+          );
+        }
+        this.taskInfo[index].completed = false;
+        localStorage.setItem('Todo', JSON.stringify(this.taskInfo));
+        window.location.reload();
+      }
+    });
+  }
+
+  removeSelectedTask(trash, index) {
+    trash.addEventListener('click', () => {
+      taskHolder.removeChild(trash.parentElement);
+      this.taskInfo.splice(index, 1);
+      this.taskInfo.forEach((list) => {
+        if (list.index > index) {
+          list.index -= 1;
+        }
+      });
+      localStorage.setItem('Todo', JSON.stringify(this.taskInfo));
+      window.location.reload();
```

Comment:
### Hello @ijosue16 
- I have a suggestion about these lines of code, I would suggest you make a function instead of the window.location.reload(). This is to avoid code repetition which makes the code DRY.